### ISSUE TITLE
Support for Normalization layer 

### ISSUE THREAD

**Issue by YuliyaCl** [Dobiasd/frugally-deep#307]
It'll be great if you add to support list Normalization layer.
The error occurs because the model weights contains not only ndarrays but also int64 element (error occured in the line 731 convert_model.py)

Code example with two similar models, but the first one will not convert to json. 

```
import tensorflow as tf
from tensorflow import keras
from keras.utils.np_utils import to_categorical
from keras.layers.core import Dense

# some input data
train_data= tf.random.uniform((10, 3))
train_labels = [False, False, False, False, False, True, True, True, True, True]
nInp = train_data.shape[1]
Y_Cat = to_categorical(train_labels, num_classes=2)    

# layers for our models
norm_layer = keras.layers.Normalization(axis=1)
layer1 = Dense(2,input_shape=(nInp,),activation='sigmoid',kernel_regularizer=keras.regularizers.l1(0.01))
layer2 = Dense(2,activation='softmax')

# model with Normalization layer
model = keras.Sequential()
model.add(keras.Input(shape=(nInp,)))
model.add(norm_layer)
model.add(layer1)
model.add(layer2)
model.compile(keras.optimizers.Adam(learning_rate=.1)) #optimization function
model.save("NN_1.h5") #converting of the file FAILED

# model without Normalization layer
model2 = keras.Sequential()
model2.add(keras.Input(shape=(nInp,)))
model2.add(layer1)
model2.add(layer2)
model2.compile(keras.optimizers.Adam(learning_rate=.1)) #optimization function
model2.fit(train_data,Y_Cat) 
model2.to_json()
model2.save("NN_2.h5") #this file wil be converted without errors
```

Thank you!

**Comment by Dobiasd** [comment]
Thanks for the idea and the good minimal example. :+1: 

When I manually circumvent the error in `convert_model.py` during the hash calculation, the C++ part, i.e., `fdeep::load_model`, (correctly) throws:

```
Loading json ... done. elapsed time: 0.000265 s
Building model ... terminate called after throwing an instance of 'std::runtime_error'
  what():  unknown layer type: Normalization
```

Adding support for  [`tf.keras.layers.Normalization` layers](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Normalization) to frugally-deep makes sense, I think.

Are you interested in giving the implementation a try in a pull request? :slightly_smiling_face:

**Comment by Dobiasd** [comment]
I interpret the lack of an answer as having no interest in implementing it. :wink:

Thus I've started working on it. :factory_worker:

**Comment by YuliyaCl** [comment]
Oh, i'm sooo sorry - i've just missed the previous answer!... 🤦

Unfortunately, I don't have any experience in C ++ and I'll be really grateful if you may solve the issue )
Thanks!

**Comment by Dobiasd** [comment]
No problem, I'll let you know (here in this issue) when done. :slightly_smiling_face:

**Comment by Dobiasd** [comment]
Done. [The newest release (`0.15.13`)](https://github.com/Dobiasd/frugally-deep/releases/tag/v0.15.13-p0) supports the [`Normalization` layer](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Normalization). :heavy_check_mark:

@YuliyaCl It would be cool if you could verify frugally-deep now being able to convert/load/use your model. :slightly_smiling_face:
